### ISSUE TITLE
[V3 Warnings] Reword docstring of allowcustomreasons

### ISSUE THREAD

**Issue by Eslyium** [Cog-Creators/Red-DiscordBot#1773]
# Feature request

#### Select the type of feature you are requesting:

- [ ] Cog
- [x] Command
- [ ] API functionality

#### Describe your requested feature
At first glance many who are unfamiliar with the bot would assume that to enable this feature they would have to type allowcustomreasons allow or disallow - this is not the case, the command will only work if you use True/False - Yes/No

![image](https://user-images.githubusercontent.com/18387809/40872129-0fc461e0-6617-11e8-9c8b-739e1daaa9fc.png)

**Comment by Tobotimus** [comment]
You're right, this is a bit misleading. Thanks for picking up on this :)

Just thought I'd note that these are the full options for `<allowed>`:
```py
('yes', 'y', 'true', 't', '1', 'enable', 'on')
('no', 'n', 'false', 'f', '0', 'disable', 'off')
```

**Comment by mborello21** [comment]
Hi! I'd like to start contributing and this looks like a good issue to start with.

Would it be okay if i took care of it?

**Comment by tekulvw** [comment]
Feel free!

**Comment by mborello21** [comment]
Thanks!

I will start looking into it then

**Comment by mborello21** [comment]
One small question. I looked a little bit at the code and i cant find where the options that Tobotimus mentioned are defined.

Could anyone point me in the right direction? Thanks in advance!

**Comment by mikeshardmind** [comment]
They aren't defined in any red specific code, but instead inherited from how discord.py handles `bool` as a typehint on command argument parsing.

Specifically : https://github.com/Rapptz/discord.py/blob/6b75179c1fed1aac0304526d130e93ad21dc84c3/discord/ext/commands/core.py#L77

**Comment by tekulvw** [comment]
Fixed with #1782
